README.md

```diff
@@ -1 +1,72 @@
-# Starting Point for Giggy
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Giggy
+
+> A Collection of some of the best jokes.
+
+<p style="display: flex; align-items: start; gap: 10px">
+  <img src="home_page_screenshot.png" width="65%" />
+  <img src="comments_popup_screenshot.png" width="32.5%" />
+</p>
+
+This is a Web Application with some jokes related to coding & Dark Humor. Created with data from the [JokeAPI](https://sv443.net/jokeapi/v2/)
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Webpack
+
+## Live Demo
+
+[Live Demo Link](https://awais-amjed.github.io/giggy/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+A Web Browser (preferably Google Chrome)
+
+### Setup
+
+- Clone the GitHub Repository
+- Go to the Project Directory
+- Run ```npm install```
+- Run ```npm build``` to build the project
+
+### Usage
+
+- Run ```npm dev``` to start the live server
```

Comment:
- [ ] Here, kindly correct the commands to `npm run build` and `npm run dev`, so the app is well documented 💪

Reply:
resolved 👍